server/src/main/java/io/crate/expression/scalar/ArrayAppendFunction.java

```diff
@@ -73,6 +79,11 @@ public final List<Object> evaluate(TransactionContext txnCtx, NodeContext nodeCt
                 resultList.add(innerType.sanitizeValue(value));
             }
         }
+        if (valueToAdd == null && calledByOperator) {
+            // array || null -> array (null is ignored)
+            return resultList;
+        }
+
```

Comment:
Can we have a test for this?

Reply:
Ah right, I was checking only new tests in this PR and in https://github.com/crate/crate/pull/16295 and missed another changes

Reply:
No worries, I should have mentioned it, it isn't obvious, sorry. ;)